Web/Edubase.Web.UI/Helpers/AuthorizedRoles.cs

```diff
@@ -13,7 +13,7 @@ public static class AuthorizedRoles
         public const string CanBulkUpdateEstablishments = EdubaseRoles.EDUBASE + "," + EdubaseRoles.EDUBASE_CMT + "," + EdubaseRoles.AP_AOS + "," + EdubaseRoles.APT + "," + EdubaseRoles.EDUBASE_CHILDRENS_CENTRE_POLICY + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.EFAHNS + "," + EdubaseRoles.FST + "," + EdubaseRoles.IEBT + "," + EdubaseRoles.SOU + "," + EdubaseRoles.EDUBASE_LACCDO + "," + EdubaseRoles.LADO + "," + EdubaseRoles.LSU + "," + EdubaseRoles.UKRLP;
         public const string CanBulkUpdateGovernors = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.ESTABLISHMENT + "," + EdubaseRoles.EDUBASE_GROUP_MAT + "," + EdubaseRoles.EFADO;
         public const string CanBulkCreateAcademies = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.AP_AOS;
-        public const string CanManageAcademyOpenings = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.AP_AOS;
+        public const string CanManageAcademyOpenings = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.AP_AOS+","+ EdubaseRoles.YCS;
```

Comment:
Reading the ticket, the requirement is to add a new link, not allow YCS to have access to the existing link:
GIVEN I'm a YCS or BO user of GIAS
WHEN selecting Tools
THEN I can view **Manage secure academy 16-19 openings**

Reply:
Sorry I think you are right.  It does not really say that in plain english on the story but based on the fact its says "secure academy 16-19 openings" I assume it is a new link.